.github/workflows/http.yml

```diff
@@ -107,22 +147,74 @@ jobs:
         shell: bash
 
       - name: Patch executable RPATH
+        if: runner.os == 'Linux'
         run: |
           patchelf --force-rpath --set-rpath '$ORIGIN/../lib' ./build/bin/${{ env.PACKAGE_NAME }}/bin/${{ env.EXEC_NAME }}
 
-      - name: Add Linux perms
+      - name: Copy shared libraries to the output directory
+        if: runner.os == 'macOS'
+        run: |
+          lipo -create -output ./build/bin/${{ env.PACKAGE_NAME }}/lib/libbreez_sdk_bindings.dylib `go list -m -f "{{.Dir}}" github.com/breez/breez-sdk-go`/breez_sdk/lib/darwin-aarch64/libbreez_sdk_bindings.dylib `go list -m -f "{{.Dir}}" github.com/breez/breez-sdk-go`/breez_sdk/lib/darwin-amd64/libbreez_sdk_bindings.dylib
+          cp `go list -m -f "{{.Dir}}" github.com/getAlby/glalby-go`/glalby/universal-macos/libglalby_bindings.dylib ./build/bin/${{ env.PACKAGE_NAME }}/lib/
+          cp `go list -m -f "{{.Dir}}" github.com/getAlby/ldk-node-go`/ldk_node/universal-macos/libldk_node.dylib ./build/bin/${{ env.PACKAGE_NAME }}/lib/
+        shell: bash
+
+      - name: Patch executable RPATH
+        if: runner.os == 'macOS'
+        run: |
+          install_name_tool -add_rpath @executable_path/../lib ./build/bin/${{ env.PACKAGE_NAME }}/bin/${{ env.EXEC_NAME }}
+
+      - name: Add permissions
         run: |
           chmod +x build/bin/${{ env.PACKAGE_NAME }}/bin/${{ env.EXEC_NAME }}
         shell: bash
 
+      - name: Sign the MacOS binary and libraries
+        if: runner.os == 'macOS'
```

Comment:
should we run this only on release, too?

Reply:
I think we need code signing as it links the dev build to our org, which then uses the previously saved org's permission to run the dev build without additional permissions again (only when the release build is run before on the device)